docs/abi-spec.rst

```diff
@@ -6,14 +6,14 @@
 Application Binary Interface Specification
 ******************************************
 
-Basic design
+Basic Design
 ============
 
 The Application Binary Interface is the standard way to interact with contracts in the Ethereum ecosystem, both
 from outside the blockchain and for contract-to-contract interaction. Data is encoded following its type,
-according to this specification.
+according to this specification.  The types are not encoded.
```

Comment:
Wouldn't "The encoding is not self describing and thus requires a schema in order to decode." be more descriptive?